tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/nonjaxrs/valid/ValidLRACSParticipant.java

```diff
@@ -78,29 +80,47 @@ public Response enlistWithCompensate(@HeaderParam(LRA.LRA_HTTP_CONTEXT_HEADER) U
     public CompletionStage<Void> compensate(URI lraId) {
         assert lraId != null;
 
+        Executor delayed = CompletableFuture.delayedExecutor(1L, TimeUnit.SECONDS);
```

Comment:
Seems like forgotten testing? Or what is the point in the delaying of the test?

Reply:
The point here is to simulate a work load and avoid false positive, but I agree it might be confusing and I can remove it.